data_model/src/query.rs

```diff
@@ -975,7 +977,34 @@ pub mod asset {
     }
 
     impl Query for FindAssetQuantityById {
-        type Output = u32;
+        type Output = AssetValue;
```

Comment:
note to sefl: update output value in docs

Reply:
who is sefl?